pkg/controller.v1beta1/experiment/experiment_controller.go

```diff
@@ -341,15 +340,24 @@ func (r *ReconcileExperiment) createTrials(instance *experimentsv1beta1.Experime
 
 	logger := log.WithValues("Experiment", types.NamespacedName{Name: instance.GetName(), Namespace: instance.GetNamespace()})
 	currentCount := int32(len(trialList))
+	logger.Info("Reconcile Suggestion", "addCount", addCount)
 	trials, err := r.ReconcileSuggestions(instance, currentCount, addCount)
 	if err != nil {
 		logger.Error(err, "Get suggestions error")
 		return err
 	}
+	var trialNames []string
 	for _, trial := range trials {
-		if err = r.createTrialInstance(instance, &trial); err != nil {
-			logger.Error(err, "Create trial instance error", "trial", trial)
-			continue
+		trialNames = append(trialNames, trial.Name)
```

Comment:
I think this paragraph can be simplified to 
```go
for _, trial := range trials {
    if err = r.createTrialInstance(instance, &trial); err != nil {
        logger.Error(err, "Create trial instance error", "trial", trial)
        continue
    }
}			
```

Reply:
In that case we try to create trials even if  `trials==[ ] `. Is that correct?

Reply:
@andreyvelich If `trials == []`,  the foreach loop will be skipped?

Reply:
@sperlingxx Thanks, I got your point. Will make change.